libs/cde-visualization/src/lib/components/node-dist-visualization/node-dist-visualization.component.scss

```diff
@@ -1,11 +1,12 @@
 :host {
   display: block;
   position: relative;
-  height: calc(100% - 8px);
   padding: 0.25rem;
   background-color: black;
   isolation: isolate;
   border-radius: 0.5rem;
+  height: 0;
```

Comment:
Won't this affect the visualization itself? It seems a bit strange to set the height to 0